### PR TITLE
Fixes issue described in #189

### PR DESCRIPTION
**Pull request by daviddq**

### CODE REVIEW

File: src/js/classes/data.js [hunk 1]
```diff
@@ -404,10 +404,10 @@ export var data = {
   },
 
   save: function() {
-    if (self.editingPath())
-      self.trySaveCurrent();
+    if (data.editingPath())
+      data.trySaveCurrent();
```
Comment (blurymind) [inline src/js/classes/data.js:408]: interesting, did self not work on electron?  :)

File: src/js/classes/input.js [hunk 2]
```diff
@@ -1,3 +1,5 @@
+import { FILETYPE } from './utils';
```
Comment (blurymind) [inline src/js/classes/input.js:1]: surprised this worked before now, we didnt have it imported

**Comment by blurymind** [comment]
Nice work :+1: 
This appears to fix it and now it behaves as expected 

Merged! :tada: 

btw I now notice the save command doesnt seem to have an icon - probably nothing to do with this pr
Screenshot_2020-05-30_13-28-13

File: src/js/classes/data.js (hunk 1 above)
Comment (daviddq) [reply src/js/classes/data.js:408]: On Electron self==window 🤷

File: src/js/classes/input.js (hunk 2 above)
Comment (daviddq) [reply src/js/classes/input.js:1]: Strangely on the webapp FILETYPE is already present in the context because _app_ already imported it I guess.

Comment (blurymind) [reply src/js/classes/input.js:1]: js is weird sometimes,isnt it :smile:

File: src/js/classes/data.js (hunk 1 above)
Comment (blurymind) [reply src/js/classes/data.js:408]: would be worth for checking in other places for stuff like this tripping electron

**Pull request merged**